tests/test_client.py

```diff
@@ -522,6 +528,42 @@ def test_apply_decision_to_order_ok(self):
             assert(response.http_status_code == 200)
             assert(response.body['entity']['type'] == 'order')
 
+    def test_apply_decision_to_session_ok(self):
```

Comment:
Just a quick question: no need to test failed cases?

Reply:
Seems that `test_apply_decision_to_session` does test with no `session_id`. I think it's enough to change the name of `test_apply_decision_to_session`.